var/spack/repos/builtin/packages/raxml-ng/package.py

```diff
@@ -21,6 +21,7 @@ class RaxmlNg(CMakePackage):
     url      = "https://github.com/amkozlov/raxml-ng/archive/1.0.1.tar.gz"
     git      = "https://github.com/amkozlov/raxml-ng.git"
 
+    version('1.0.2', submodules=True)
```

Comment:
Small OCD, but for v1.0.1 we have a string instead of a Boolean. Can you fix that too?

Reply:
See @adamjstewart comment below, they should both be Boolean